node/resolvers/Queries/Users.ts

```diff
@@ -135,6 +135,63 @@ export const getAllUsersByEmail = async (_: any, params: any, ctx: Context) => {
   return getAllUsers({ masterdata, logger, where })
 }
 
+export const getUsersByEmailPaginated = async ({
+  masterdata,
+  logger,
+  email,
+  page = 1,
+  pageSize = 25,
+}: {
+  masterdata: any
+  logger: any
+  email: string
+  page: number
+  pageSize: number
+}) => {
+  try {
+    const users = [] as any[]
+
+    const resp = await masterdata.searchDocumentsWithPaginationInfo({
+      dataEntity: config.name,
+      fields: [
+        'id',
+        'roleId',
+        'clId',
+        'email',
+        'name',
+        'orgId',
+        'costId',
+        'userId',
+        'canImpersonate',
+        'active',
+      ],
+      pagination: {
+        page,
+        pageSize,
+      },
+      schema: config.version,
+      where: `email = "${email}"`,
+    })
+
+    const { data } = resp as unknown as {
+      pagination: {
+        total: number
+      }
+      data: any
+    }
+
+    users.push(...data)
+
+    return users
+  } catch (error) {
+    logger.error({
+      error,
+      message: 'Profiles.getUsersByEmailPaginated-error',
+    })
+    throw new Error(error)
+  }
+}
+
```

Comment:
This function was added here but it is not used anywhere else. Also no graphql schema was defined for it and it is not mentioned anywhere in the PR.
Can we remove it? If needed, lets add it in another PR.

Reply:
Hi @enzomerca 
It was used for a test and I forgot to remove. I've removed the method now.